tests/languages/sas/macrodefinition_feature.test

```diff
@@ -0,0 +1,44 @@
+%macro prnt(var,sum);
+  proc print data=srhigh;
+    var &var;
+    sum &sum;
+  run;
+%mend prnt;
+
+----------------------------------------------------
+
+[
+  [
+    "macro-declaration",
+    [
+      ["keyword", "%macro"],
+      " prnt(var,sum);"
+    ]
+  ],
+  ["step", "proc print"],
+  ["keyword", "data"],
+  ["operator", "="],
+  "srhigh",
+  ["punctuation", ";"],
+  ["keyword", "var"],
+  ["operator", "&"],
+  "var",
+  ["punctuation", ";"],
+  ["keyword", "sum"],
+  ["operator", "&"],
+  "sum",
+  ["punctuation", ";"],
+  ["step", "run"],
+  ["punctuation", ";"],
+  [
+    "macro-end",
+    [
+      ["keyword", "%mend"],
+      " prnt;"

```

Comment:
The `;` is supposed to be punctuation, right?

Reply:
Yes, I'll add a punctuation rule to macro-end and macro-declaration.

Reply:
There are quite a few tokens which have the same problem. Instead of including the `;` in the token, we could just use a lookahead to not include it and use the global punctuation to match the `;`.

Reply:
I've addressed this.